meta/running_lunch_and_learn.md

```diff
@@ -0,0 +1,51 @@
+# Running a Lunch & Learn
+
+Hello Artsy employee or curious outsider! This document outlines how to run our [lunch and learns][lal]. The process of planning a L&L spans weeks and generally goes through the following steps. The planning process is managed in the [Lunch and Learns GitHub project](https://github.com/artsy/meta/projects/1).
+
+* **Ideas**: These can people we'd like to present or ideas that we'd like to find speakers for – either Artsy employees or external speakers we'd like to invite in.
+* **In Conversation**: We have reached out and are in talks with someone specific to present a L&L.
+* **Scheduled**: A date has been confirmed with the speaker and they have received a calendar event invitation.
+* **Add to README/Dropbox/Youtube**: The presentation is finished but needs to be documented ([see below][after]).
+* **Done**: It's done!
+
+Read on for more specifics.
+
+## Handling a Referral
+
+Sometimes you'll get an introduction to a speaker from a colleague, passing the baton to you to organize the particulars. No problem – just follow these steps:
+
+* Say thanks!
+* Pass along the link to the [lunch_and_learn.md][lnl].
+* Add a card to the "In Conversation" section of the Lunch and Learn project, or move the existing card there from "Ideas".
+* Discuss what topic makes sense.
+* Pick a date and move their card to the "Scheduled" column (and add the date to the card title). Send them a calendar invitation.
+
+## Running a Lunch & Learn
+
+### The Monday Before
+
+Send a note to Camille and Katherine to be included in the weekly newsletter. Make some noise on Slack to get folks excited to come on Thursday.
+
+### The Day Before
+
+* Double-check with the speaker that they're still available. Make sure to thank them again for their time.
+* Add the speaker to [Envoy](https://dashboard.envoy.com).
+
+### The Day Of
+
+Be on 26 at 11:40 to receive our guest. Take them upstairs and while they're getting set up, offer some water. Get them set up on the Zoom.us meeting and remind them that you'll take them out for lunch afterward.
+
+At 11:50, send an `@here` reminder in the #dev (or #general, depending). Give folks a few minutes past noon to show up, but don't hold the show up waiting for them.
+
+Immediately before you present the speaker, hit the record button in Zoom.us.
+
+### Afterward
+
+Right after the talk, ask the speaker for their slides. Then take the speaker and a few folks out to lunch. Tataki is never a bad idea. You can bring a few others who attended if there were any keeners. Make sure to expense this meal.
```

Comment:
> were any keeners

?

Reply:
I didn't realize this was so regional, but "keener" [means](http://english.entangled.com/what-is-a-keener/):

> Keener is a Canadian slang term to refer to someone who is keen (eager) to learn

I have it there to prevent twenty people trying to join lunch 😄 I'll reword.